node/runtime/src/lib.rs

```diff
@@ -166,7 +166,7 @@ impl balances::Trait for Runtime {
 }
 
 parameter_types! {
-	pub const MinimumPeriod: u64 = SECS_PER_BLOCK / 2;
+	pub const MinimumPeriod: u64 = SLOT_DURATION / 2;
```

Comment:
not your change but historically wrong: this should be
```suggestion
	pub const MinimumPeriod: Moment = SLOT_DURATION / 2;
```

```
pub trait Trait: system::Trait {
	/// Type used for expressing timestamp.
	type Moment: Parameter + Default + SimpleArithmetic + ... 
...
	type MinimumPeriod: Get<Self::Moment>;
}
```